cli/command/trust/signer_add.go

```diff
@@ -31,7 +31,7 @@ type signerAddOptions struct {
 func newSignerAddCommand(dockerCli command.Cli) *cobra.Command {
 	var options signerAddOptions
 	cmd := &cobra.Command{
-		Use:   "add OPTIONS NAME REPOSITORY [REPOSITORY...] ",
+		Use:   "add [OPTIONS] NAME REPOSITORY [REPOSITORY...] ",
```

Comment:
Wondering if the brackets were omitted because `--key` is a required option 🤔

Reply:
you're right, my bad 😅